src/DotNetStatus/Common/AnalyzerInfo.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+namespace Microsoft.Fx.Portability
+{
+    public class AnalyzerInfo
+    {
+        public string AnalyzerName { get; set; }
+        public string Id { get; set; }
+        public ICollection<CompatibilityRangeAttribute> Attributes { get; set; }
+        public bool IsCompatibilityDiagnostic { get; set; }
+
+        public int IdNumber
+        {
+            get
+            {
+                var matches = Regex.Match(Id, @"(\d+)");
```

Comment:
Start up performance issues are fine if they are a one-time thing (it's a website, so they are long-running). Can you switch it to a method, though, instead of a property? Another option is to find the range of characters that would be the number and do a .SubString(start, length)